lib/lighthouse/benefits_claims/service.rb

```diff
@@ -162,24 +163,27 @@ def validate526(body, lighthouse_client_id = nil, lighthouse_rsa_key_path = nil,
 
     private
 
-    def build_request_body(body)
+    def build_request_body(body, transaction_id = "vagov-#{SecureRandom}")
       body = body.as_json
       if body.dig('data', 'attributes').nil?
         body = {
           data: {
             type: 'form/526',
             attributes: body
+          },
+          meta: {
+            transaction_id:
           }
         }
       end
       body.as_json.deep_transform_keys { |k| k.camelize(:lower) }
     end
 
-    def prepare_submission_body(body)
+    def prepare_submission_body(body, transaction_id)
       # if we're coming straight from the transformation service without
       # making this a jsonapi request body first ({data: {type:, attributes}}),
```

Comment:
Should `({data: {type:, attributes}})` be updated to include the new format that includes the addition of `meta: { transaction_id: }` to the request body? `attributes` also appears to be be missing a colon at the end.

Reply:
drat. great point! i'll address this in refactor. thank you, @freeheeling